src/pages/PrintOnlyScreen.tsx

```diff
@@ -71,11 +89,22 @@ const PrintOnlyScreen = ({
     if (isUsed) {
       await printer.print()
       updateTally()
+      setBallotId('')
```

Comment:
Won’t this just immediately generate a new one because of your conditional after the `useState` initializer?

Reply:
no longer relevant given Beau's PR, but the idea here is yeah to say "done with this ballot Id" and have the earlier conditional regenerate a new one the next time it's rendered for the next ballot.